CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+## [5.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v4.4.0...v5.0.0) (2019-01-03)
+
+## BREAKING CHANGES 💣
+
+We dropped some legacy stuff related to Redux. Please refer to the PR [#197](https://github.com/commercetools/merchant-center-application-kit/pull/197) to get more detailed info and to follow the upgrade migration guide.
+
+Furthermore, we dropped support to the following deprecated things:
+
+- format of permissions as objects (`[{ mode: 'manage', resource: 'products' }]`) from the `@commercetools-frontend/permissions` package
+- components to access user and project data `<FetchUser>` and `<FetchProject>` from the `@commercetools-frontend/application-shell` package
+
+#### 🐛 Type: Bug
+
+- `application-shell`
+  - [#216](https://github.com/commercetools/merchant-center-application-kit/pull/216) Fix test utils and quick access ([@dferber90](https://github.com/dferber90))
+- `application-shell`
+  - [#211](https://github.com/commercetools/merchant-center-application-kit/pull/211) fix(login): reset password dialog redirect, refactor to use formik ([@emmenko](https://github.com/emmenko))
+
+#### 💥 Type: Breaking Change
+
+- `application-shell`, `permissions`, `sdk`
+  - [#242](https://github.com/commercetools/merchant-center-application-kit/pull/242) chore: drop deprecated components ([@emmenko](https://github.com/emmenko))
+- `application-shell`, `permissions`
+  - [#217](https://github.com/commercetools/merchant-center-application-kit/pull/217) Refactor `Authorized` in permissions pkg to not rely on constants ([@tdeekens](https://github.com/tdeekens))
```

Comment:
This is very unlikely breaking but rather a feature. Anyway better safe than sorry.

Reply:
Ah because it had the "breaking change" label...it's also listed in "Refactoring". Should I remove this point from "breaking change"?

Reply:
Yeah I think it makes sense. Sorry it was two PRs in one. Should have done two. We really should get better at small PRs at times :). Also the lodash refactor could be a separate entry.

Reply:
Yes agree 👌

> Also the lodash refactor could be a separate entry

Ah good point, we can add an entry for that yes.